dbms/src/DataStreams/HashJoinProbeBlockInputStream.h

```diff
@@ -20,6 +20,25 @@
 namespace DB
 {
 
+class SquashingHashJoinBlockTransform
```

Comment:
move to a separate file and add unit test for it

Reply:
> move to a separate file and add unit test for it

done.